### PR TITLE
Added back-end support for settings issues resolved

### LINKED ISSUE #7: Implement a working Settings activity
# Description
Add a settings/preference system to the app. For now, it should be able to set the FaceAccess server hostname/IP and the port as per user preference.

# Resources
These resources should provide you with a good starting point to begin with.

1. https://codelabs.developers.google.com/codelabs/android-training-adding-settings-to-app/index.html?index=..%2F..android-training#0
2. https://blog.mindorks.com/implementing-preferences-settings-screen-in-android

# Other Advisories
Fork the project and create another branch from the `development` to work on this feature.

### PR DESCRIPTION
**Pull request by bagofcodes**
#7

### CODE REVIEW

File: app/src/main/java/io/github/dotslash21/faclient/AuthActivity.java [hunk 1]
```diff
@@ -42,6 +41,8 @@
 import io.github.dotslash21.faclient.utils.GraphicOverlay;
 import io.github.dotslash21.faclient.utils.FaceIdentificationProcessor;
 
+import static android.hardware.Camera.*;
```
Comment (dotslash21) [inline app/src/main/java/io/github/dotslash21/faclient/AuthActivity.java]: What is the need for this?? `import android.hardware.Camera` worked fine..

File: app/src/main/res/values/styles.xml [hunk 2]
```diff
@@ -8,7 +8,7 @@
         <item name="colorAccent">@color/colorAccent</item>
     </style>
 
-    <!-- The launcher theme. It sets the main window background to the launch_screen drawable -->
+<!--     The launcher theme. It sets the main window background to the launch_screen drawable-->
```
Comment (dotslash21) [inline app/src/main/res/values/styles.xml:11]: Unnecessary change

**Review by dotslash21 — CHANGES_REQUESTED** [review]

**Comment by bagofcodes** [comment]
Uncomment hoe gechilo tai

File: app/src/main/java/io/github/dotslash21/faclient/AuthActivity.java (hunk 1 above)
Comment (bagofcodes) [reply app/src/main/java/io/github/dotslash21/faclient/AuthActivity.java]: resolved

**Review by dotslash21 — APPROVED** [review]

**Comment by dotslash21** [comment]
Good job!
